### ISSUE TITLE
Error loading WebsiteDemo project

### ISSUE THREAD

**Issue by pfm-solita** [digst/OIOSAML.Net#73]
When opening the solution I get an error about creating a virtual directory and the WebsiteDemo project does not load.
Not sure if this is a problem on my machine only.
Changing IISUrl in the project file from https://oiosaml-net.dk:20002/ to https://localhost:20002/ solved it for me.

**Comment by ronnieholm** [comment]
Sounds familiar. I believe if you delete the IISExpress auto-created directory within the solution and relaunch WebsiteDemo, the problem goes away. It's a general VS + IISExpress issue.

**Comment by pfm-solita** [comment]
Not sure what directory to delete.
If i clone the repository into a new folder and open the solution with Visual Studio 2022 (v 17.7.3) I get this dialog.
![image](https://github.com/digst/OIOSAML.Net/assets/141002900/232a3257-3455-41a6-b51c-d104e6557a5d)

IdentityProviderDemo.csproj is being loaded and has <IISUrl>https://localhost:20001</IISUrl>.
WebsiteDemo.csproj is not loaded and has <IISUrl>https://oiosaml-net.dk:20002/</IISUrl>.

If the current configurations of IISUrl in the project files are correct, feel free to close this issue, as it may just be an issue on my machine.

**Comment by pfm-solita** [comment]
I can see how changing the hostname to localhost might be a problem if NemLog-in is expecting the user to be redirected from and back to oiosaml-net.dk. So changing the configuration is probably not the right solution.

**Comment by pfm-solita** [comment]
If other people run into this issue, they way I solved is was to change IISUrl in WebsiteDemo.csproj to use the hostname localhost instead of oiosaml-net.dk. Then I opened the solution in Visual Studio and closed it again. This will create .vs\dk.nita.saml20\config\applicationhost.config. In that file I then found the site-configuration for WebsiteDemo and changed the hostname in the bindings from localhost to oiosaml-net.dk. I also changed IISUrl back in WebsiteDemo.csproj. I can then open the solution again and the WebsiteDemo project loads correctly.
